tests/test_evaluable.py

```diff
@@ -684,18 +684,19 @@ class asciitree(TestCase):
   @unittest.skipIf(sys.version_info < (3, 6), 'test requires dicts maintaining insertion order')
   def test_asciitree(self):
     f = evaluable.Sin(evaluable.Inflate(1, evaluable.Zeros((), int), 2)**evaluable.Diagonalize(evaluable.Argument('arg', (2,))))
+    self.maxDiff = None
```

Comment:
@gertjanvanzwieten What comment should I put here?

Reply:
something to keep us from accidentally deleting it, like `# keep assertEqual from shortening its arguments`